1-exercises/D-for-in-loops/exercise-2.js

```diff
@@ -24,6 +24,8 @@ let shapes = {
 let shapesWithMoreThanFiveSides = [];
 
 // WRITE YOUR CODE BELOW THIS LINE
+for (const shape in shapes)
+if (shapes[shape] <= 5) shapesWithMoreThanFiveSides.push(shape);
```

Comment:
Good, Wrong operator here though so it doesn't produce the desired result of constructing an array of shapes which have 5 or more sides - it constructs an array with shapes with 5 or less sides